pkg/skaffold/schema/defaults/defaults.go

```diff
@@ -127,6 +127,9 @@ func SetDefaultRenderer(c *latestV2.SkaffoldConfig) {
 	if c.Render.Generate.Helm != nil {
 		return
 	}
+	if c.Deploy.HelmDeploy != nil {
+		return
+	}
```

Comment:
should we rename `HelmDeploy` to `LegacyHelmDeploy` in the  schema `config.go`?

Reply:
Was meaning to ask about this. Since there won't be anything that replaces HelmDeploy, I thought it would be okay to leave as is. Thoughts?

Reply:
update: discussed this offline, we will move to calling things `LegacyHelmDeploy` to give clear indication that we will no longer be actively supporting feature development of the helm deployer